### PR TITLE
php: make httpd a build time dependency.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
This is only needed in the test (which will now fail) and we'll make it `:test` when https://github.com/Homebrew/homebrew-test-bot/pull/144 and https://github.com/Homebrew/brew/pull/3875 are merged and in a new tag.

### CODE REVIEW

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Review by SMillerDev — APPROVED** [review]

**Review by ilovezfs — APPROVED** [review]
:+1: when 🍏

**Comment by ilovezfs** [comment]
May this bring peace and joy to many.

**Comment by alcaeus** [comment]
Thank you @MikeMcQuaid and @ilovezfs! ❤️

**Comment by ihorvorotnov** [comment]
Wohooo! Awesome! Thanks to everyone who made it happen :)

File: Formula/php.rb [hunk 1]
```diff
@@ -3,13 +3,17 @@ class Php < Formula
...
 
   bottle do
     sha256 "39be15089755343b860c293f9c0c54d93f15185059ef5b6d539270355b41978d" => :high_sierra
     sha256 "0bd906b8e2f7a52476e3cdfedf8db7d74cf27b1aad05db2793035ac918fd723a" => :sierra
     sha256 "50b051315cb30e71020675f47559a4f6f75edc45b3150c7d82ad4778d5e1132b" => :el_capitan
   end
 
+  depends_on "apr" => :build
+  depends_on "apr-util" => :build
+  depends_on "httpd" => :build
```
Comment (ilovezfs) [inline Formula/php.rb]: needs => :test

**Pull request closed without merging**